ui/src/app/views/account-creation-success/component.tsx

```diff
@@ -0,0 +1,166 @@
+import {Component, DoCheck, Input, OnInit} from '@angular/core';
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {
+  BolderHeader,
+  Header,
+  SmallHeader
+} from 'app/components/headers';
+import {AccountCreationComponent} from 'app/views/account-creation/component';
+import {
+  AccountCreationResendModalReact,
+  AccountCreationUpdateModalReact
+} from 'app/views/account-creation-modals/component';
+import {LoginComponent} from 'app/views/login/component';
+
+
+const styles = {
+  buttonLinkStyling: {
+    border: 'none',
+    cursor: 'pointer',
+    outlineColor: 'transparent',
+    color: '#2691D0',
+    backgroundColor: 'transparent',
+  }
+};
+
+interface AccountCreationSuccessProps {
+  username: string;
+  contactEmailOnCreation: string;
+  creationNonce: string;
+}
+
+interface AccountCreationSuccessState {
+  resendModal: boolean;
+  updateModal: boolean;
+  contactEmail: string;
+}
+
+export class AccountCreationSuccessReact
+    extends React.Component<AccountCreationSuccessProps, AccountCreationSuccessState> {
+  state: AccountCreationSuccessState;
+  props: AccountCreationSuccessProps;
+
+  constructor(props: AccountCreationSuccessProps) {
+    super(props);
+    this.state = {
+      contactEmail: this.props.contactEmailOnCreation,
+      resendModal: false,
+      updateModal: false,
+    };
+  }
+
+  public closeModals(): void {
+    this.setState({resendModal: false});
+    this.setState({updateModal: false});
```

Comment:
In practice, only one of these will be in effect at a time. We could break this apart and inline each one into its relevant spot below.